### PR TITLE
Feature/button

### PR DESCRIPTION
**Pull request by kingawr123**
- Basic Button component 
- some test
- storybook

### CODE REVIEW

File: tests/components/Button.test.tsx [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+    fireEvent.click(btn);
+    expect(onClick).toHaveBeenCalledTimes(1);
+  });
+
+  test("disable button doesn't fire onClick event", () => {
+    render(<Button isDisable={true} onClick={onClick}>Test</Button>);
+    const disableBtn  = screen.getByRole('button');
+
+    fireEvent.click(disableBtn);
+    expect(onClick).toHaveBeenCalledTimes(0);
```
Comment (michalszc) [inline tests/components/Button.test.tsx:32]: Replace with .not.toBeCalled()

File: stories/components/Button.stories.tsx [hunk 2]
```diff
@@ -1,60 +1,87 @@
 import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
+
```
Comment (michalszc) [inline stories/components/Button.stories.tsx:3]: Maybe without this empty line?

File: stories/components/Button.stories.tsx [hunk 3]
```diff
@@ -1,60 +1,87 @@
...
+    isDisable: {
+      defaultValue: false,
+      control: 'boolean',
+    }
+  } 
 } as ComponentMeta<typeof Button>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args}/>;
```
Comment (michalszc) [inline stories/components/Button.stories.tsx:27]:
```suggestion
const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
```

**Review by michalszc — CHANGES_REQUESTED** [review]

**Review by kingawr123 — COMMENTED** [review]
i've made the requested changes

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simpleui",
-  "version": "0.0.1",
+  "version": "0.1.0",
```
Comment (michalszc) [inline package.json:3]:
```suggestion
  "version": "0.2.0",
```

**Review by michalszc — CHANGES_REQUESTED** [review]

**Review by michalszc — APPROVED** [review]

**Pull request merged**